sql-cli/sql_cli/connections.py

```diff
@@ -40,15 +26,25 @@ def _create_or_replace_connection(conn_obj: Connection) -> None:
         session.commit()
 
 
-def validate_connections(environment: str = "default", connection_id: str | None = None) -> None:
+def validate_connections(
+    project: Project, environment: str = "default", connection_id: str | None = None
+) -> None:
     """
     Validates that the given connections are valid and registers them to Airflow with replace policy for existing
     connections.
     """
     config_file_contains_connection = False
-    connections = _load_yaml_connections(environment)
+
+    airflow_meta_conn = retrieve_airflow_meta_database_conn(project.directory / project.airflow_home)
+    os.environ["AIRFLOW__DATABASE__SQL_ALCHEMY_CONN"] = airflow_meta_conn
+
+    importlib.reload(airflow)
+    importlib.reload(airflow.configuration)
+    importlib.reload(airflow.models.base)
+    importlib.reload(airflow.models.connection)
```

Comment:
We should add documentation around this as it seems somewhat hacky. Also do we need to put this in its own function? Will this be required for other functions like run?

Reply:
Done, @dimberman ! I'm not sure if run will need this, but I refactored and isolated this, trying to add some comments.